core/state/trie_prefetcher.go

```diff
@@ -362,6 +354,7 @@ func newSubfetcher(db Database, root common.Hash, accountHash common.Hash) *subf
 
 // schedule adds a batch of trie keys to the queue to prefetch.
 func (sf *subfetcher) schedule(keys [][]byte) {
+	atomic.AddUint32(&sf.pendingSize, uint32(len(keys)))
```

Comment:
Why need to change the location of this line?

Reply:
`pendingSize` is added on `schedule`, while decreased in `loop`, move it ahead to avoid decreasing before increasing.

Reply:
`pendingSize` is added on `schedule`, while decreased in `loop`, move it ahead to avoid decreasing before increasing.